src/styles.css

```diff
@@ -4,7 +4,7 @@ body {
 }
 
 .header {
-  height: 50px;
+  height: 49px;
```

Comment:
Your comment doesn't match the fix. It says it was 51 but should be 50, but it was 50 and now it's 49.